nodes/nomos-node/Cargo.toml

```diff
@@ -26,7 +26,7 @@ nomos-http = { path = "../../nomos-services/http", features = ["http"] }
 nomos-consensus = { path = "../../nomos-services/consensus", features = ["libp2p"] }
 nomos-storage = { path = "../../nomos-services/storage", features = ["sled"] }
 nomos-libp2p = { path = "../../nomos-libp2p" }
-nomos-da = { path = "../../nomos-services/data-availability" }
+nomos-da = { path = "../../nomos-services/data-availability", features = ["libp2p"] }
```

Comment:
Added this because it was not possible to build `nomos-node` with `cargo build -p nomos-node`.